docs/dictionary/command/save.lcdoc

```diff
@@ -23,21 +23,29 @@ save this stack as "Backup"
 Example:
 save stack "project1" as "/Disk/Folder/File"
 
+Example:
+save stack "MyOldApp" with format 7.0
+
 Parameters:
 stack: Any open stack.
+
 filePath: The filePath specifies the name and location of the file you want to save to. If you specify a name but not a location, LiveCode assumes the file is in the defaultFolder. If the file does not exist, LiveCode creates it.
 
-The result: If the stack cannot be saved (for example, if you try to save it to a nonexistent drive or to a CD-ROM), the result <function> is set to "Can't open stack file".
+stackFormat: The <stack version|stack file format version> to use when saving.
+
+The result: If the stack cannot be saved (for example, if you try to save it to a nonexistent drive or to a CD-ROM), the <result> will contain an error message.  Otherwise, the <result> will be empty.
 
 Description:
 Use the <save> <command> to save changes to a <stack> or to save a copy of a <stack> into another <file>.
 
 The <save> <command> saves all <stacks> stored in the same <file> as the specified <stack>. That is, if you save a <main stack>, all <substacks> of that stack are also saved in the same <file>, and if you save a <substack>, its <main stack> and any other <substacks> are also saved.
 
-If the stack has not yet been saved and doesn't have a filename, you must use theform.
+If the stack has not yet been saved and doesn't have a filename, you must use the <as> form.
+
+If you specify a <stack version> to be used when saving, but the <stack> cannot be saved in that format without losing data, <save> will fail.
```

Comment:
Am I wrong or is this only the case for widgets? If so perhaps that should be clarified.